include/nnstreamer-edge.h

```diff
@@ -96,58 +96,199 @@ typedef void (*nns_edge_data_destroy_cb) (void *data);
 
 /**
  * @brief Create edge handle.
+ * @param[in] id Unique id in local network
+ * @param[in] connect_type value of nns_edge_flag_e. Connection type between edge nodes.
+ * @param[in] flags value of @a nns_edge_flag_e. The value can be set by using bitwise-or operation.
+ * @param[out] edge_h The edge handle. edge handle should be released using nns_edge_release_handle().
+ * @return 0 on success. Otherwise a negative error value.
+ * @retval #NNS_EDGE_ERROR_NONE Successful.
+ * @retval #NNS_EDGE_ERROR_NOT_SUPPORTED Not supported.
+ * @retval #NNS_EDGE_ERROR_OUT_OF_MEMORY Failed to allocate required memory.
+ * @retval #NNS_EDGE_ERROR_INVALID_PARAMETER Given parameter is invalid.
+ *
+ * Here is an example of the usage:
+ * @code
+ * static int _nns_edge_event_cb (nns_edge_event_h event_h, void *user_data) {
+ *   nns_edge_event_e event_type;
+ *   nns_edge_event_get_type (event_h, &event_type);
+ *   switch (event_type) {
+ *     case NNS_EDGE_EVENT_CAPABILITY:
+ *     case NNS_EDGE_EVENT_NEW_DATA_RECEIVED:
```

Comment:
recommend to add functions - nns_edge_event_parse_new_data() and nns_edge_event_parse_capability()
```
case NNS_EDGE_EVENT_CAPABILITY:
{
  char *capability;
  nns_edge_event_parse_capability (event_h, &capability);
  /** 
   * Check capability string and return NNS_EDGE_ERROR_NONE if capability is acceptable.
   * Returning error code will deny the connection.
   */
}
case NNS_EDGE_EVENT_NEW_DATA_RECEIVED:
{
  nns_edge_data_h data_h;
  nns_edge_event_parse_new_data (event_h, &data_h);
  /**
   * You should release returned data_h using nns_edge_data_destroy().
   * Recommend not to spend too much time in event callback.
   */
}
```

Reply:
Thanks. Updated.